src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs

```diff
@@ -155,6 +156,11 @@ private bool BeforeExecuteCommand(bool speculative, TypeCharCommandArgs args, Co
                 delimiters = startingNode.GetBrackets();
             }
 
+            if (delimiters == default && TryGetParentsWithBraces(startingNode, out var parentsWithBraces))
+            {
+                delimiters = parentsWithBraces.GetBraces();
+            }
```

Comment:
Why this is different than above code that calculates brackets and parentheses? The above just calls `startingNode.GetX`

Reply:
Changed : )